src/controller/CommissioningWindowOpener.h

```diff
@@ -107,6 +107,39 @@ class CommissioningWindowOpener
                                        Callback::Callback<OnOpenCommissioningWindow> * callback, SetupPayload & payload,
                                        bool readVIDPIDAttributes = false);
 
+    /**
+     * @brief
+     *   Try to look up the bridged device on the bridge with the given node id
```

Comment:
None of this has anything to do with bridges, as far as this class is concerned.  The right thing to document is "send an OpenCommissioningWindow command with the given parameters to the given endpoint of the given node" or something along those lines.

The fact that this endpoint happens to be a bridge thing of some sort is irrelevant to this class.